raster/r.category/main.c

```diff
@@ -103,9 +108,22 @@ int main(int argc, char *argv[])
     parm.fmt_coeff->description =
         _("Two pairs of category multiplier and offsets, for $1 and $2");
 
+    parm.format = G_define_standard_option(G_OPT_F_FORMAT);
+    parm.format->key = "output_format";
+    parm.format->guisection = _("Print");
+
```

Comment:
What is the name of the variables you used in the other implementations? Because having the variable named format not being assigned with the input of `format`, but rather `output_format`, is a source of confusion (doesn't reflect the design intent well).
It isn't a real conflict as such as, format is assigned to `fmt`.

Reply:
Ya I saw. So, consistency with code of other modules is respectable and justifiable. It follows more closely another engineering concept, of following design patterns so it is easily understood as they are ubiquitous. So it is fine like this ;)

Reply:
Understood.

Reply:
Do we have an issue tracking our wishes for a next major release? If yes, we should add it now while we remember it, if not, create one (using it like a work item).